src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,115 @@
+import React from 'react';
+import { ERROR, FilterBy } from '../../types/enums';
+import { Todo } from '../../types/Todo';
+import { deleteTodo } from '../../api/todos';
+
+type Props = {
+  todos: Todo[];
+  filterQwery: FilterBy;
+  setFilterQwery: (qwery: FilterBy) => void;
+  setErrorMessage: (value: ERROR) => void;
+  setTodos: (callback: (prev: Todo[]) => Todo[]) => void;
+  todosLoading: number[];
+  loading: {
```

Comment:
```suggestion
  loadingIdsState: {
```
bad naming or idsToLoadState smth like this